build.sbt

```diff
@@ -213,6 +213,31 @@ lazy val contribs = (project in file("contribs"))
     Compile / compile := ((Compile / compile) dependsOn createTargetClassesDir).value
   ).configureUnidoc()
 
+lazy val deltaSharing = (project in file("deltasharing"))
```

Comment:
lets just name the directory sharing. the whole repo is already delta... so does not make sense to have a subdir also named "delta*"

Reply:
updated.